packages/fluentui/react-northstar/src/components/Portal/PortalInner.tsx

```diff
@@ -28,40 +29,39 @@ export interface PortalInnerProps extends ChildrenComponentProps {
 /**
  * A PortalInner is a container for Portal's content.
  */
-class PortalInner extends React.Component<PortalInnerProps> {
-  static contextType = PortalBoxContext;
-
-  static propTypes = {
-    ...commonPropTypes.createCommon({
-      accessibility: false,
-      as: false,
-      className: false,
-      content: false,
-      styled: false,
-    }),
-    mountNode: PropTypes.object,
-    onMount: PropTypes.func,
-    onUnmount: PropTypes.func,
-  };
+const PortalInner: React.FC<PortalInnerProps> & FluentComponentStaticProps<PortalInnerProps> = props => {
+  const context = React.useContext(PortalBoxContext);
+  const { children, mountNode } = props;
 
-  componentDidMount() {
-    _.invoke(this.props, 'onMount', this.props);
-  }
+  // PortalInner should render elements even without a context
+  // eslint-disable-next-line
+  const target: HTMLElement | null = isBrowser() ? context || document.body : null;
+  const container: HTMLElement | null = mountNode || target;
+  React.useEffect(() => {
+    _.invoke(props, 'onMount', props);
 
-  componentWillUnmount() {
-    _.invoke(this.props, 'onUnmount', this.props);
-  }
+    return () => _.invoke(props, 'onUnmount', props);
+  });
```

Comment:
```suggestion
  }, []);
```

This effect should be executed once